tutorial.html

```diff
@@ -36,7 +36,7 @@ <h3>Running the Model</h3>
     <h3>Producing Vector Output</h3>
     <p>Next we'll polygonize the output to produce a vector file. We'll use the polygonize command for this.</p>
     <pre><code>ftw inference polygonize austria-inf.tif</code></pre>
-    <p>This will create a GeoParquet file named <code>austria-inf.parquet</code> in the current directory. It will follow the <a href="https://github.com/fieldsoftheworld/fiboa">Fiboa standard</a> for geospatial data, and include the area. The polygonization algorithm is not very complex - right now it simplifies things to 15 meters, which smoothes the edges, and removes any polygons that are under 500 meters (configurable with the <code>--simplify</code> flag). We hope to offer more polygonization options in the future, and <a href="contributing.html">contributions are welcome</a>. You can also generate GeoJSON, FlatGeobuf or GeoPackage output, just use the <code>--output</code> flag and end the file name with the appropriate extension.</p>
+    <p>This will create a GeoParquet file named <code>austria-inf.parquet</code> in the current directory. It will follow the <a href="https://github.com//fiboa">Fiboa standard</a> for geospatial data, and include the area. The polygonization algorithm is not very complex - right now it simplifies things to 15 meters, which smoothes the edges, and removes any polygons that are under 500 meters (configurable with the <code>--simplify</code> flag). We hope to offer more polygonization options in the future, and <a href="contributing.html">contributions are welcome</a>. You can also generate GeoJSON, FlatGeobuf or GeoPackage output, just use the <code>--output</code> flag and end the file name with the appropriate extension.</p>
```

Comment:
```suggestion
    <p>This will create a GeoParquet file named <code>austria-inf.parquet</code> in the current directory. It will follow the <a href="https://github.com/fiboa">Fiboa standard</a> for geospatial data, and include the area. The polygonization algorithm is not very complex - right now it simplifies things to 15 meters, which smoothes the edges, and removes any polygons that are under 500 meters (configurable with the <code>--simplify</code> flag). We hope to offer more polygonization options in the future, and <a href="contributing.html">contributions are welcome</a>. You can also generate GeoJSON, FlatGeobuf or GeoPackage output, just use the <code>--output</code> flag and end the file name with the appropriate extension.</p>
```